bedrock/firefox/templates/firefox/choose.html

```diff
@@ -0,0 +1,58 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "base-resp.html" %}
```

Comment:
Since this is in the `/firefox/` app should it be extending `firefox/base-resp.html`?